tests/codegen/waiter-tests/model/waiter-operations.smithy

```diff
@@ -0,0 +1,73 @@
+namespace com.test
+
+use aws.protocols#restJson1
+use smithy.waiters#waitable
+
+service Lambda {
```

Comment:
nit

I'd choose a unique service name, it makes grepping and debugging easier when something fails and only one thing is named that. e.g. `WaiterTestService` or whatever